opwen_email_server/mailers/registry.py

```diff
@@ -0,0 +1,7 @@
+from opwen_email_server.mailers.echo.format import EchoEmailFormatter
+
+ECHO_ADDRESS = 'echo@bot.lokole.ca'
```

Comment:
Better to import this from `opwen_email_server.mailers.echo.format` so that we don't duplicate the value and have to remember to update it in two places in the future.

Reply:
Will do!